run.ps1

```diff
@@ -124,10 +124,10 @@ function Get-KoreBuild {
             }
 			 
             # Hack to use Core version
-            Write-Host "!!! Hack to use .NET Core SDK 5.0.301"
```

Comment:
Shall we upgrade to latest versions? See version information: https://github.com/dotnet/installer.